using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -2,68 +2,248 @@
 
 ## Events handled
 
-The following events are handled by the Identity extension:
+### Identity request identity
 
-### Hub shared state
+This event is used for the Identity extension in one of the following situations:
 
-The shared state event is dispatched by the event hub after a shared state is created or updated. The event data contains the event owner and the name of the module to which the shared state belongs. This event is generated by the Event Hub.
+- To retrieve the custom visitor identifiers
+- To retrieve the Experience Cloud ID (MID)
+- To append the visitor identifiers to an URL
+- To synchronize a visitor identifier or a list of identifiers with the Adobe Experience Cloud ID Service
 
-### Data payload definition   <a id="data-payload-definition"></a>
+This event is generated by:
+
+- Calling the Identity `getIdentifiers` API
+- Calling the Identity `getExperienceCloudId` API
+- Calling the Identity `appendVisitorInfoForURL` (Android) / `appendToUrl` (iOS) API
+- Calling the Identity `getUrlVariables` API
+- Calling the Identity `syncIdentifier` or `syncIdentifiers` APIs
+
+#### Event details
+
+| **Event Type**               | **Event Source**                      | **Paired** | **Paired Event**                                             |
+| :--------------------------- | :------------------------------------ | :--------- | :----------------------------------------------------------- |
+| com.adobe.eventType.identity | com.adobe.eventSource.requestIdentity | Yes        | [Identity Response Identity Event](#identity-response-identity) |
+
+#### Data payload definition
 
 Here are the key-value pairs in this event:
 
-| **Key** | **Value Type** | **Optional** | **Description** |
-| :--- | :--- | :--- | :--- |
-| `stateowner` | String | N/A | N/A |
+| **Key**               | **Value Type**      | **Optional** | **Description**                                              |
```

Comment:
Some of the sentences in the "Description" column are missing periods; please add as appropriate.